packages/shared/src/graphql/fragments.ts

```diff
@@ -65,6 +65,7 @@ export const SOURCE_BASE_FRAGMENT = gql`
     membersCount
     currentMember {
       role
+      roleRank
       referralToken
       permissions
     }
```

Comment:
Will have some issue with current fragment changes

Reply:
I'll check if there is any conflict and coordinate with Ante if necessary.